### DISCUSSION TITLE
Hello.

### DISCUSSION THREAD

**Discussion by itsmicky777** [termux/termux-app#2908] [Q&A]
why when i try to install firebase it gives this error?
![Screenshot_20220725-223159.png](https://user-images.githubusercontent.com/94807428/180984569-5c765e93-7cde-4c9f-a99a-7708d5160680.png)

**Comment by agnostic-apollo** [comment]
Do not use external storage `storage/`/`/sdcard` for development. Use `$HOME`/`~/`.

https://wiki.termux.com/wiki/Internal_and_external_storage

**Reply by itsmicky777** [reply]
What? Can you explain a little more? I'm a bit dumb and I don't really know how it works, I'm a "beginner"
